packages/slice-machine/package.json

```diff
@@ -38,8 +38,7 @@
   "dependencies": {
     "@babel/register": "^7.11.5",
     "@reach/menu-button": "^0.10.5",
-    "@slicemachine/core": "^1.0.0",
-    "@svgr/webpack": "^5.4.0",
+    "@slicemachine/core": "^1.0.0",    "@svgr/webpack": "^5.4.0",
```

Comment:
just some kind of linter issue i guess

Reply:
can you put this dependency at a new line like all the others :)